packages/php-wasm/universal/src/lib/ini.ts

```diff
@@ -0,0 +1,44 @@
+import { PHP_INI_PATH } from './base-php';
+import { UniversalPHP } from './universal-php';
+import { stringify, parse } from 'ini';
+
+export async function getPhpIniEntries(php: UniversalPHP, entries?: string[]) {
+	const ini = parse(await php.readFileAsText(PHP_INI_PATH));
+	if (entries === undefined) {
+		return ini;
+	}
+	const result: Record<string, unknown> = {};
+	for (const key of entries) {
+		result[key] = ini[key];
+	}
+	return result;
+}
+
+export async function setPhpIniEntries(
+	php: UniversalPHP,
+	entries: Record<string, unknown>
+) {
+	const ini = parse(await php.readFileAsText(PHP_INI_PATH));
+	for (const [key, value] of Object.entries(entries)) {
+		if (value === undefined || value === null) {
+			delete ini[key];
+		} else {
+			ini[key] = value;
+		}
+	}
+	await php.writeFile(PHP_INI_PATH, stringify(ini));
+}
+
+export async function withPHPIniValues(
```

Comment:
This could use some comments. The name didn't make it clear to me that this runs a single callback with a set of custom PHP ini values.

Reply:
Good call! I'll document that in a follow-up PR.